fabfile/add.py

```diff
@@ -156,3 +156,10 @@ def tpch(**kwargs):
 def jdbc():
     'Adds everything required to test out the jdbc connecter'
     sudo('yum install -q -y java-1.6.0-openjdk-devel') # we only need java on the master
+
+@task
+@roles('master')
+def add_coordinator():
+    utils.psql("SELECT master_add_node('localhost', {}, groupid => 0);".format(config.PORT))
+    utils.psql("SELECT 1 FROM master_set_node_property('localhost', {}, 'shouldhaveshards', true);"
```

Comment:
might be nice if this was a separate option, since we typically don't put shards on the coordinator